media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayer = window.dataLayer = window.dataLayer || [];
 
-        dataLayer.push({
+    function sendCoreDataLayer() {
+        var dataLayerCore = {
             'event': 'core-datalayer-loaded',
-            'pageId': Mozilla.Analytics.getPageId()
-        });
+            'pageId': Mozilla.Analytics.getPageId(),
+            'pageHasDownload': Mozilla.Analytics.pageHasDownload(),
+            'pageHasVideo': Mozilla.Analytics.pageHasVideo(),
+            'pageVersion': Mozilla.Analytics.getPageVersion(),
+            'testPilotUser': window.navigator.testpilotAddon ? window.navigator.testpilotAddon : 'false'
```

Comment:
The reason I'm using a string and not a Boolean is that Google converts Boolean to 0 or 1 which doesn't make for a great user experience in the GA reports. So instead I always will pass the strings 'true' or 'false' and not Boolean values.